tests/snappi_tests/multidut/pfcwd/files/pfcwd_multidut_basic_helper.py

```diff
@@ -304,7 +305,12 @@ def __gen_traffic(testbed_config,
             data_flow.tx_rx.port.tx_name = tx_port_name
             data_flow.tx_rx.port.rx_name = rx_port_name
 
-            eth, ipv4 = data_flow.packet.ethernet().ipv4()
+            eth, ipv4, udp = data_flow.packet.ethernet().ipv4().udp()
+            src_port = random.randint(5000, 6000)
+            udp.src_port.increment.start = src_port
+            udp.src_port.increment.step = 1
+            udp.src_port.increment.count = 2
```

Comment:
Previously: 400G port speed1 traffic stream at 100%
Now: 400G port speed 2 traffic streams at 50% each

Reply:
@amitpawar12 @vmittal-msft : FYI

Reply:
@rraghav-cisco pls make it platform specific.